src/libecalc/presentation/yaml/model.py

```diff
@@ -71,7 +71,7 @@ def _create_yaml_configuration(main_yaml_path: Path) -> PyYamlYamlModel:
     def _read_resource(resource_name: Path, *args, read_func: Callable[..., Resource]):
         try:
             return read_func(resource_name, *args)
-        except ValueError as exc:
+        except (InvalidResourceHeaderException, ValueError) as exc:
```

Comment:
yes, here since you "trust" InvalidResourceHeaderException (its our exception), and we have a user-friendly message, you can show the message to the user, e.g. add it to the "failed to read message"